Makefile.smartview

```diff
@@ -0,0 +1,103 @@
+.PHONY: all pull fetch patch combine generate clean codegen mod docs move-other patch-post fmt test stage validate
+
+include Makefile
+
+PACKAGE_NAME=smartview
+# OAS2:
+SOURCE_URLS=https://developer.equinix.com/sites/default/files/smartview-v2-catalog-environmental_v2_0.json \
+            https://developer.equinix.com/sites/default/files/2024-09/power_v1.json \
+			https://developer.equinix.com/sites/default/files/2024-09/environmental_v1.json \
+			https://developer.equinix.com/sites/default/files/smartview-v2-catalog-feed-subscription_v2_3.json \
+			https://developer.equinix.com/sites/default/files/smartview-v1-catalog-alert_v1_0.yaml \
+			https://developer.equinix.com/sites/default/files/smartview-v1-catalog-assets_v1_1.json \
+			https://developer.equinix.com/sites/default/files/smartview-v1-catalog-hierarchy_v1_2.json			
+# TODO(OAS3): https://developer.equinix.com/sites/default/files/smartview-v2-catalog-system-alerts_v2_2.json
+SPEC_ROOT_FILE=swagger.yaml
+
+CODE_DIR=${CODE_BASE_DIR}/${PACKAGE_NAME}
+TEMPLATE_DIR=${TEMPLATE_BASE_DIR}/${PACKAGE_NAME}
+SPEC_FETCHED_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/oas3.fetched
+SPEC_PATCH_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/patches
+SPEC_PATCHED_DIR=${SPEC_BASE_DIR}/${PACKAGE_NAME}/oas3.patched
+MERGED_SPEC_FILE=${SPEC_BASE_DIR}/${PACKAGE_NAME}/merged-spec.yaml
+SPEC_FETCHER=curl
+
+all: pull fetch patch combine generate stage
+
+generate: clean codegen remove-unused patch-post mod fmt test
+
+pull:
+	${CRI} pull ${OPENAPI_IMAGE}
+
+# Fetch multiple source URLs into SPEC_FETCHED_DIR with numeric suffixes
+fetch:
+	mkdir -p ${SPEC_FETCHED_DIR}
+	rm -f ${SPEC_FETCHED_DIR}/*
+	$(eval INDEX=0)
+	for URL in ${SOURCE_URLS}; do \
+		${SPEC_FETCHER} \
```

Comment:
For whatever reason, these `curl` requests are failing. I believe this was a problem we had back in early versions of metal-go that was worked-around by adding a meaningful User-Agent. No such luck this time.